src/mw/render/vk/scene.cpp

```diff
@@ -14,6 +14,7 @@ using namespace imp;
 namespace render {
 namespace vk {
 
+#if 0
```

Comment:
Let's just dump all the RT specific dead code into some RT file that doesn't get compiled.